packages/cohere/src/cohere-chat-language-model.ts

```diff
@@ -298,9 +303,19 @@ export class CohereChatLanguageModel implements LanguageModelV1 {
                 return;
               }
 
+              case 'tool-plan-delta': {
+                toolPlan += value.delta.message.tool_plan;
+                return;
+              }
+
```

Comment:
can you handle it like a regular text delta and immediately send (instead of keeping a variable and sending all at once)?

would simplify things and lead to better streaming. is the anything here that I'm missing?

Reply:
Good call, done.